taxa/taxonomy/taxoneditor.php

```diff
@@ -77,7 +77,7 @@
 <!DOCTYPE html>
 <html lang="<?php echo $LANG_TAG ?>">
 <head>
-	<title><?php echo $DEFAULT_TITLE." ".(isset($LANG['TAX_EDITOR'])?$LANG['TAX_EDITOR']:'Taxon Editor').": ".$tid; ?></title>
+	<title><?php echo $DEFAULT_TITLE . " " . $LANG['TAX_EDITOR'] . ": " . $tid; ?></title>
```

Comment:
Can this be combined into one line, so e.g., "Taxon Editor: Animalia [6]" ?
![image](https://github.com/BioKIC/Symbiota/assets/16688784/54e9b1b0-dbca-480b-b631-9a1ab6872ac5)

Reply:
(Oops, this refers to the h1 below, not the title)

Reply:
I think I fixed this in f81fcc778.